lib/activerecord-import/active_record/adapters/postgresql_adapter.rb

```diff
@@ -3,5 +3,15 @@
 
 class ActiveRecord::ConnectionAdapters::PostgreSQLAdapter
   include ActiveRecord::Import::PostgreSQLAdapter
+
+  alias_method :post_sql_statements_orig, :post_sql_statements
+  def post_sql_statements( table_name, options ) # :nodoc:
+    unless options[:primary_key].blank?
+      post_sql_statements_orig(table_name, options) << (" RETURNING #{options[:primary_key]}")
+    else
+      post_sql_statements_orig(table_name, options)
+    end
+  end
```

Comment:
The main win I see of using the module is that we get all the benefits we seek and we don't end up having to adding any additional names to the PostgreSQLAdapter (for which we don't own). 

Basically, trying to reduce name pollution and potential collision as much as possible. I admit it's unlikely that the aliased method name collides (especially if `post_sql_statement` isn't conflicting after all this time), but if we can remove the chance altogether, why not?